pkgs/by-name/gs/gsc/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gsc";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "https://git.launchpad.net/gsc";
+    sha256 = "sha256-U5lP8tpk8Vfsg2GMC80Az/oHdmqHApYZFX01clT/moo=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  preConfigure = "substituteInPlace gsc.c --replace '/usr/share/GSC' '${placeholder "out"}/share/GSC'";
```

Comment:
Could you use a multi-line string for this?

Reply:
Sure, I forfgot to run nixfmt...